Add new option to remove task by ID

Add a new menu option to remove a task by its integer id. This should update the console menu, add a handler method, and implement `TodoList.remove(int id)` to return true if a task is removed and false otherwise.

### Acceptance criteria

* Menu shows 4) Remove by id (and Quit becomes 5) Quit).
* `TodoApp.removeTask()` reads an id and prints [INFO] Removed. or [ERROR] Not found.
* `TodoList.remove(int id)` removes by id and returns boolean.

--- a/TodoApp.java
+++ b/TodoApp.java
@@ -10,7 +10,8 @@ public class TodoApp {
         System.out.println("=== TODO (Simple) ===");
         while (running) {
             showMenu();
-            if (!scanner.hasNextLine()) break;
+            if (!scanner.hasNextLine())
+                break;
             String choice = scanner.nextLine().trim();
             handleChoice(choice);
         }
@@ -22,7 +23,8 @@ public class TodoApp {
         System.out.println("1) Add task");
         System.out.println("2) List tasks");
         System.out.println("3) Mark DONE by id");
-        System.out.println("4) Quit");
+        System.out.println("4) Remove by id");
+        System.out.println("5) Quit");
         System.out.print("Choose: ");
     }
 
@@ -34,6 +36,8 @@ public class TodoApp {
         } else if ("3".equals(choice)) {
             markDone();
         } else if ("4".equals(choice)) {
+            removeTask();
+        } else if ("5".equals(choice)) {
             running = false;
         } else {
             System.out.println("[ERROR] Invalid option.");
@@ -47,6 +51,14 @@ public class TodoApp {
         String desc = scanner.nextLine();
         int id = todoList.add(title, desc);
         System.out.println("[INFO] Added task id " + id);
+    }
+
+    private void removeTask() {
+        System.out.print("Enter id to remove: ");
+        String s = scanner.nextLine();
+        int id = Integer.parseInt(s);
+        boolean ok = todoList.remove(id);
+        System.out.println(ok ? "[INFO] Removed." : "[ERROR] Not found.");
     }
 
     private void listTasks() {

--- a/TodoApp.java
+++ b/TodoApp.java
@@ -56,9 +56,14 @@ public class TodoApp {
     private void removeTask() {
         System.out.print("Enter id to remove: ");
         String s = scanner.nextLine();
-        int id = Integer.parseInt(s);
-        boolean ok = todoList.remove(id);
-        System.out.println(ok ? "[INFO] Removed." : "[ERROR] Not found.");
+        try {
+            int id = Integer.parseInt(s);
+            boolean ok = todoList.remove(id);
+            System.out.println(ok ? "[INFO] Removed." : "[ERROR] Not found.");
+        } catch (NumberFormatException e) {
+            System.out.println("[ERROR] Please enter a numeric id.");
+        }
+
     }
 
     private void listTasks() {

--- a/TodoList.java
+++ b/TodoList.java
@@ -11,6 +11,16 @@ public class TodoList {
         return t.getId();
     }
 
+    public boolean remove(int id) {
+        for (int i = 0; i < items.size(); i++) {
+            if (items.get(i).getId() == id) {
+                items.remove(i);
+                return true; // immediate return to avoids index issues
+            }
+        }
+        return false;
+    }
+
     public boolean markDone(int id) {
         for (int i = 0; i < items.size(); i++) {
             if (items.get(i).getId() == id) {

